### ISSUE TITLE
Fails to detect relative links?

### ISSUE THREAD

**Issue by bebraw** [bem-site/broken-links-checker#8]
I have links like `[previous post](../the-story-so-far)` at my site. That page uses relative link and works online. For some reason this gets flagged as a false positive and I get

```
WARN lib/checker.js: Broken [404] link: => ../the-story-so-far on page: => http://localhost:8000/blog/balance
```

**Comment by tormozz48** [comment]
Hello @bebraw 

Thank you for report this issue.

Unfortunately I can not give you answer at this moment.
But I will check my tests and maybe will create a stub site structure to verify this case.

**Comment by veged** [comment]
Closing this as part of archiving this repository. Link validation for bem.info now belongs to the main site CI: bem-site/bem.info#88.
